### PR TITLE
#743 Allow users to determine if a twisty panel is open or not by default

### PR DESCRIPTION
**Pull request by matthoward366**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/harness/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@elyra/canvas": "file:../common-canvas",
     "@hot-loader/react-dom": "17.0.0-rc.2",
     "@ibm/plex": "5.1.3",
-    "autoprefixer": "10.0.0",
+    "autoprefixer": "10.3.1",
```
Comment (matthoward366) [inline canvas_modules/harness/package.json:40]: These changes are for a security warning showing up in github

File: canvas_modules/common-canvas/src/common-properties/panels/twisty/twisty.scss [hunk 2]
```diff
@@ -18,10 +18,6 @@
 	&.hide {
 		display: none;
 	}
-	&[disabled] {
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/panels/twisty/twisty.scss:21]: Using carbon's twisty disable option instead.

File: canvas_modules/common-canvas/src/common-properties/panels/action-panel/action-panel.scss [hunk 3]
```diff
@@ -17,7 +17,7 @@
 .properties-action-panel {
 	display: flex;
 	flex-wrap: wrap;
-	margin: 3px 0;
+	padding-bottom: $spacing-03;
```
Comment (matthoward366) [inline canvas_modules/common-canvas/src/common-properties/panels/action-panel/action-panel.scss:20]: There are some minor updates to action panel/buttons spacing.

File: canvas_modules/common-canvas/__tests__/common-properties/form/form-test.js [hunk 4]
```diff
@@ -165,7 +166,7 @@ describe("Correct form should be created", () => {
 		expect(isEqual(JSON.parse(JSON.stringify(placementResource.expectedResult)), JSON.parse(JSON.stringify(generatedForm)))).to.be.true;
 	});
 
-	it("should create a form with actions and summaryPanel", () => {
+	it.only("should create a form with actions and summaryPanel", () => {
```
Comment (caritaou) [inline canvas_modules/common-canvas/__tests__/common-properties/form/form-test.js:169]: Forgot to remove the `.only`

Comment (matthoward366) [reply canvas_modules/common-canvas/__tests__/common-properties/form/form-test.js]: Good catch.  Fixed.

**Review by caritaou — APPROVED** [review]

**Pull request merged**